addons/website/models/digest.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_website_visitor_cnt = fields.Boolean('Visitors')
+    kpi_website_visitor_cnt_value = fields.Integer(compute='_compute_kpi_website_visitors_cnt_value')
+    kpi_tracked_page_view_cnt = fields.Boolean('Tracked page views')  # Non uniq
```

Comment:
``kpi_website_track_count`` in order to match the actual model name ?

Reply:
Yes good idea, it is changed